app/controllers/games_controller.rb

```diff
@@ -3,11 +3,12 @@ class GamesController < ApplicationController
 
   def index
     @games = Game.all
```

Comment:
@phearn I'll need to implement the Game.avail back into this after we approve this PR. We had a bit of scope creep going on, so I'm trying to determine what's been implemented and what has not.